website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -16,6 +16,10 @@ dbt Cloud release notes for recent and historical changes. Release notes fall in
 
 Release notes are grouped by month for both multi-tenant and virtual private cloud (VPC) environments.
 
+## March 2025
+
+- **New:** The dbt Cloud-native integration with Azure DevOps now supports [Entra ID service principals](/docs/cloud/git/setup-service-principal). Unlike a services user, which represents a real user object in Entra ID, the service principal is a secure identity associated with your dbt Cloud app and used to access resources in Azure unattended. Please [migrate your service user](/docs/cloud/git/setup-service-principal#migrate-to-service-principal) for Azure DevOps to a service principal as soon as possible.
```

Comment:
what does 'unattended' mean here? not sure i understand that on my end?
```suggestion
- **New:** The dbt Cloud-native integration with Azure DevOps now supports [Entra ID service principals](/docs/cloud/git/setup-service-principal). Unlike a services user, which represents a real user object in Entra ID, the service principal is a secure identity associated with your dbt Cloud app to access resources in Azure unattended. Please [migrate your service user](/docs/cloud/git/setup-service-principal#migrate-to-service-principal) to a service principal for Azure DevOps  as soon as possible.
```

Reply:
Unattended access means that there is no user intervention. With a service user account, it's a user object and needs to be treated like any other user, meaning if there are password expiry rules, or in this case, forced MFA, someone needs to log in as that user and update passwords or provide them elsewhere. With unattended access, there's no need to manage any of that.